jacdac-ble/JacdacBLE.cpp

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
don't prepend with ``jacdac-``, just ``ble`` or bgetter ``bluetooth`` to stay consistent.

Reply:
I prepended jacdac because we might actually want to have a bluetooth service in addition to a bluetooth transport. These are separate. One is a service, the other is a "physical layer".

Reply:
folder != package name

Reply:
Bluetooth-transport

Reply:
ok will change to `bluetooth-transport`?